ios/FluentUI/Tab Bar/TabBarItemView.swift

```diff
@@ -173,6 +186,33 @@ class TabBarItemView: UIControl {
         }
     }
 
+    @objc private func isUnreadValueDidChange() {
+        isUnreadDotVisible = item.isUnread
+        updateUnreadDot()
+        setNeedsLayout()
```

Comment:
do we need to update accessibilityLabel what we are going to say if there is an unread dot?

Reply:
None of the other controls with unread dots currently support accessibilityLabel functionality. Because of this, we do not have a localized string we can use for this label right now. 

Do you want me to add an "unread" label in the unread dot case for now and get a loc pass done? Happy to chat offline about how to do this.

Separately, we should update SegmentedControl and PillButtonBar to have accessibilityLabels for their unreadDots.

Reply:
I've gone ahead and updated the loc file with an unread dot string. Tested with accessibility inspector.

Reply:
we should follow whatever apple does for app badges on home screen. would need to figure out if it is part of acessibilitylabel or accessibilityhint

Reply:
The current badge logic updates the accessibilityLabel. I will follow that pattern and if we need to update it to use accessibilityHint instead, I think we should address that in another change for both badge and unread dot, to keep this change constrained.